_viash.yaml

```diff
@@ -1,16 +1,64 @@
 viash_version: 0.9.0-RC6
 
+# Step 1: Change the name of the task.
+# example: task_name_of_this_task
 name: task_template
+organization: openproblems-bio
+version: dev
+# Step 2: Update the description to a short description of the task.
 description: |
   An OpenProblems benchmark task.
 license: MIT
+# Step 3: Add keywords to describe the task.
 keywords: [single-cell, openproblems, benchmark]
+# Step 4: Update the `task_template` to the name of the task from step 1.
 links:
-  issue_tracker: https://github.com/openproblems-bio/task_<task_name>/issues
-  repository: https://github.com/openproblems-bio/task_<task_name>
-  docker_registry: ghcr.io/openproblems-bio
+  issue_tracker: https://github.com/openproblems-bio/task_template/issues
+  repository: https://github.com/openproblems-bio/task_template
+  docker_registry: ghcr.io
+# Step 5: Update the info fields to the text from the task issue. 
+info:
+  # A unique identifier. Can only contain lowercase letters, numbers or underscores. (This is the same as the name above but without the `task_` prefix)
+  name: template
+  # A unique, human-readable, short label. Used for creating summary tables and visualisations.
+  label: Template
+  description: |
+    Provide a clear and concise description of your task, detailing the specific problem it aims
+    to solve. Outline the input data types, the expected output, and any assumptions or constraints.
+    Be sure to explain any terminology or concepts that are essential for understanding the task.
+  summary: A one sentence summary of purpose and methodology. Used for creating an overview tables.
+  motivation: |
+    Explain the motivation behind your proposed task. Describe the biological or computational 
+    problem you aim to address and why it’s important. Discuss the current state of research in
+    this area and any gaps or challenges that your task could help address. This section 
+    should convince readers of the significance and relevance of your task.
+  image: The name of the image file to use for the component on the website.
+  # Step 6: Replace the task_template to the name of the task in `info.name`.
+  test_resources:
+    - type: s3
+      path: s3://openproblems-data/resources_test/task_template/
+      dest: resources_test/task_template
+    - type: s3
+      path: s3://openproblems-data/resources_test/common/
+      dest: resources_test/common
 
-version: dev
+# Step 7: Update te authors of the task.
+authors: 
+  # Full name of the author, usually in the name of FirstName MiddleName LastName.
+  - name: Kai Waldrant 
```

Comment:
This is a good way to automatically become the author and maintainer of every new benchmark! ;)